services-js/registry-certs/client/birth/questions/DateOfBirth.tsx

```diff
@@ -6,36 +6,39 @@ import { TextInput } from '@cityofboston/react-fleet';
 import QuestionComponent from './QuestionComponent';
 import FieldsetComponent from './FieldsetComponent';
 
+import { researchFeeDisclosureText } from '../../common/FeeDisclosures';
+
 interface Props {
   forSelf: boolean | null;
   firstName: string;
-  dateOfBirth?: string;
+  birthDate: string;
 
   handleProceed: (answers: State) => void;
   handleStepBack: () => void;
 }
 
 interface State {
-  dateOfBirth: string;
+  birthDate: string;
 }
 
+// todo: this component will be a datepicker
```

Comment:
TBH a next field with a placeholder and maybe even the fancy auto-slash thing would be fine by me.

Reply:
I’m leaving this as-is because it hasn’t yet been decided what sort of field this will be. My rec has been to use `<input type="date" />` with a fallback [month dropdown][day dropdown][year text field]... but I didn’t want to spend any time on a solution that might be scrapped.

Reply:
👍